go/billing/tests/helpers.py

```diff
@@ -74,7 +93,7 @@ def mk_transaction(account, tag_pool_name='pool1',
 
 def mk_statement(account,
                  title=settings.MONTHLY_STATEMENT_TITLE,
-                 type=Statement.TYPE_MONTHLY,
+                 _type=Statement.TYPE_MONTHLY,
```

Comment:
Trying to avoid using a reserved word here, and went with the convention that twisted uses (if I'm remembering correctly) since that seems to be what we are most familiar with. Will go with `statement_type` if you prefer that.